samples/certs/generate-workload.sh

```diff
@@ -35,15 +44,21 @@ subjectAltName = critical, @alt_names
 URI = $san
 EOF
 
-openssl req -new -key "workload-$name-key.pem" -subj "/" -out workload.csr -config workload.cfg
+openssl req -new -key "$dir/$tmp/workload-$name-key.pem" -subj "/" -out "$dir"/workload.csr -config "$dir"/workload.cfg
 
-openssl x509 -req -in workload.csr -CA ca-cert.pem -CAkey ca-key.pem -CAcreateserial \
--out "workload-$name-cert.pem" -days 3650 -extensions v3_req -extfile workload.cfg
+openssl x509 -req -in "$dir"/workload.csr -CA "$dir"/ca-cert.pem -CAkey "$dir"/ca-key.pem -CAcreateserial \
+-out "$dir/$tmp/workload-$name-cert.pem" -days 3650 -extensions v3_req -extfile "$dir"/workload.cfg
 
-cat cert-chain.pem >> "workload-$name-cert.pem"
+cat "$dir"/cert-chain.pem >> "$dir/$tmp/workload-$name-cert.pem"
 
 echo "Generated workload-$name-[cert|key].pem with URI SAN $san"
-openssl verify -CAfile <(cat cert-chain.pem root-cert.pem) "workload-$name-cert.pem"
+openssl verify -CAfile <(cat "$dir"/cert-chain.pem "$dir"/root-cert.pem) "$dir/$tmp/workload-$name-cert.pem"
 
 # clean temporary files
-rm ca-cert.srl workload.cfg workload.csr
+if [ -f "$dir"/.srl ]; then
```

Comment:
Just need rm -rf ${TMP_DIR} if all generated files are in tmp dir.

Reply:
The tmp dir only contains the temp certs, on purposely keep other files still in current dir which not impacts original script. And the tmp dir would need to be cleaned up in go code after the test execution.